README.md

```diff
@@ -161,6 +314,250 @@ more metric, you must run CASTanet for the first metric
+
+_ARGUMENTS:_
+  cast_dict: A dictionary of files and corresponding CASTs
+
+_RETURNS:_
+  dict: files and the corresponding amount of comments
+
+#### count_whileloops
+
+Count the number of while loops in a Python file.
+
+`counter.count_whileloops`
+
+_ARGUMENTS:_
+  cast_dict: A dictionary of files and corresponding CASTs
+
+_RETURNS:_
+  dict: files and the corresponding amount of while loops
+
+#### count_forloops
+
+Count the number of for loops in a Python file.
+
+`counter.count_for_loops`
+
+_ARGUMENTS:_
+  cast_dict: A dictionary of files and corresponding CASTs
+
+_RETURNS:_
+  dict: files and the corresponding amount of for loops
+
+#### match_if_statements
+
+Count the number of if statements in a Python file.
+
+`counter.match_if_statements(cast_dict)`
+
+_ARGUMENTS:_
+  cast_dict: A dictionary of files and corresponding CASTs
+
+_RETURNS:_
+  dict: files and the corresponding amount of if statements
+
+#### match_func_defs
+
+Count the number of function definitions in a Python file.
+
+`counter.match_funcdefs(cast_dict)`
+
+_ARGUMENTS:_
+  cast_dict: A dictionary of files and corresponding CASTs
+
+_RETURNS:_
+  dict: files and the corresponding amount of function_definitions
+
+#### count_function_without_docstrings
+
+Count the number of functions without docstrings.
+
+`counter.count_function_without_docstrings(func_count)`
+
+_ARGUMENTS:_
+  func_count (dict): A dictionary of functions and docstring counts per file
+
+_RETURNS:_
+  int: total number of functions - total number of docstrings
+
+#### exists_docstring
+
+Determine if a docstring exists for a specified function.
+
+`counter.exists_docstring(cast_dict, function_name)`
+
+_ARGUMENTS:_
+  cast_dict: A dictionary of files and corresponding CASTs
+  function_name (str): Name of function to check for docstrings
+
+_RETURNS:_
+  -1: function does not exist
+  0: function exists without docstring
+  1: function exists with docstring
+
+#### match_class_def
+
+Count the number of class definitions in a Python file.
+
+`counter.match_class_defs(cast_dict)`
+
+_ARGUMENTS:_
+  cast_dict: A dictionary of files and corresponding CASTs
+
+_RETURNS:_
+  dict: files and the corresponding amount of class definitions
+
+#### count_class_defs_without_docstrings
+
+Count the number of class definitions without docstrings.
+
+`counter.count_class_defs_without_docstrings(class_count)`
+
+_ARGUMENTS:_
+  class_count (dict): A dictionary of classes and docstring counts per file
+
+_RETURNS:_
+  int: total number of classes - total number of docstrings
+
+#### count_function_arguments
+
+Count the number of arguments for a specified function.
+
+_ARGUMENTS:_
+  cast_dict: A dictionary of files and the corresponding CASTs
+  function_name: User-specified file of interest
+
+_RETURNS:_
+  -1: Function was not found
+  else: The amount of parameters for the given function
+
+#### assignment_count
+
+Count the number of assignment statements in a Python file.
```

Comment:
Do these assignment statements include augmented assignments? If so, that fact should be specified in this documentation, especially because there is another function to count those aug assignments.

Reply:
Bailey